src/Administration/Resources/app/administration/src/app/component/form/select/entity/sw-entity-many-to-many-select/index.js

```diff
@@ -0,0 +1,497 @@
+/**
+ * @sw-package framework
+ */
+
+import template from './sw-entity-many-to-many-select.html.twig';
+
+const { Component } = Shopware;
+const { debounce, get } = Shopware.Utils;
+const { deepCopyObject } = Shopware.Utils.object;
+const { Criteria, EntityCollection } = Shopware.Data;
+
+/**
+ * @final
```

Comment:
Apparently we cannot use `final` here, at least the tests are failing, but there should be something that plugins can use the components
```suggestion
 * @private
```

Reply:
please leave it `@private` and already add a deprecation, e.g. `@deprecated tag:v6.8.0 - will be removed, use "sw-entity-multi-select" instead`

Reply:
```suggestion
 * @private
 * @deprecated tag:v6.8.0 - will be removed, use `sw-entity-multi-select` instead
 
```